src/parse-result.js

```diff
@@ -57,14 +57,23 @@ export function namespace(options) {
     }
   }
 
-  class SourceMap extends minim.BaseElement {
+  class SourceMap extends minim.Element {
     constructor(...args) {
       super(...args);
       this.element = 'sourceMap';
     }
+
+    // Override toValue because until Refract 1.0
+    // sourceMap is special element that contains array of array
+    // TODO Remove in next minor release
```

Comment:
Since the content isn't a true array element but array of non-element (array). This means that the base classes `toValue` implementation doesn't handle this special case in sourceMap.

This change would be temporarily IMO to be able to make this release updating minim across the board. In the next minor release of minim the deserialisation rules will change so that the content will be an array element and thus the default `toValue` implementation logic would work and this override can be removed.

```
content = [
  [
    NumberElement(..),
    NumberElement(..),
  ]
]
```